src/serializer/ResidualHeapVisitor.js

```diff
@@ -101,13 +113,9 @@ export class ResidualHeapVisitor {
     }
 
     // visit properties
-    for (let [key, propertyBinding] of obj.properties) {
+    for (let [, propertyBinding] of obj.properties) {
```

Comment:
`for (let propertyBinding of obj.properties.values())` is probably more efficient.